src/main.py

```
@@ -35,68 +35,75 @@
             console.print(f"[yellow]Removed {dupes} duplicate feeds[/yellow]")
-        
+
         # Process feeds
         with Progress() as progress:
-            task = progress.add_task("[cyan]Analyzing feeds...", total=len(manager.feeds))
-            
+            task = progress.add_task(
+                "[cyan]Analyzing feeds...", total=len(manager.feeds)
+            )
+
             for feed_hash, feed in manager.feeds.items():
```

Ruff (0.8.0)

Loop control variable `feed_hash` not used within loop body

(B007)